python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -878,6 +860,8 @@ def _process_auto_materialize_tick_generator(
 
         yield error_info
 
+    _process_auto_materialize_tick = materialize(_process_auto_materialize_tick_generator)
```

Comment:
pardon my python ignorance - does this still pass "self" through in the way you would expect? I assume many tests would be failing if it did not

Reply:
Yep; self is just yet another argument!